bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/report/job/AnalyzeJob.java

```diff
@@ -39,78 +46,98 @@ private AnalyzeJob(String name) {
+        AtomicInteger taskCount = new AtomicInteger(0);
+        if (fileToAnalyze != null) {
+            createSkipFile();
+            taskCount.incrementAndGet();
+        } else {
+            // read the file
+            try {
+                Files.lines(logFile).forEach(new Consumer<String>() {
 
-                @Override
-                public void accept(String t) {
-                    if (t.contains("\"command\":")) {
-                        taskCount.incrementAndGet();
+                    @Override
+                    public void accept(String t) {
+                        if (t.contains("\"command\":")) {
+                            taskCount.incrementAndGet();
+                        }
                     }
-                }
-            });
-        } catch (IOException e) {
-            Logger.log(IStatus.ERROR, "Couldn't read logFile!");
+                });
+            } catch (IOException e) {
+                Logger.log(IStatus.ERROR, "Couldn't read logFile!");
+                if (projectView != null)
+                    projectView.refresAsync(projectView::setEmptyMsg);
+            }
         }
 
         monitor.beginTask("Starting Analysis...", taskCount.get() * 2);
         try {
             // TODO make numberOfAnalyzers a parameter depending on the turned on analyzer
             // engines.
             int numberOfAnalyzers = 2;
-            config.getCodeChecker().analyze(logFile, true, monitor, taskCount.get() * numberOfAnalyzers, config);
+            config.getCodeChecker().analyze(logFile, true, monitor, taskCount.get() * numberOfAnalyzers, config,
+                    skipFile);
         } catch (NullPointerException e) {
             Logger.log(IStatus.ERROR, "Could not complete the analysis");
+            if (projectView != null)
+                projectView.refresAsync(projectView::setEmptyMsg);
         }
-
-        deleteLogFile();
+        if (projectView != null)
+            projectView.refresAsync(projectView::setEmptyMsg);
         return Status.OK_STATUS;
     }
     
-    @Override
-    protected void canceling() {
-        config.getCodeChecker().cancelAnalyze();
-    }
-
-    /**
-     * Creates a copy of the log file created by ld logger, to avoid concurrency
-     * issues.
-     * 
-     * @throws IOException
-     *             Thrown when the copying fails.
-     */
-    private void copyLogFile() throws IOException {
-        Path originalLogFile = project.getLogFileLocation();
-        logFile = Paths.get(originalLogFile.toAbsolutePath().toString() + System.nanoTime());
-        Files.copy(originalLogFile, logFile);
-    }
-
-    /**
-     * Delete the temporary logfile after the analysis.
-     */
-    public void deleteLogFile() {
+    public void createSkipFile() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("+").append(fileToAnalyze.toAbsolutePath().toString()).append(System.lineSeparator());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "+" appears 5 times in the file.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641621)